java/code/src/com/suse/manager/model/hub/HubFactory.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2024 SUSE LLC
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ */
+package com.suse.manager.model.hub;
+
+import com.redhat.rhn.common.hibernate.HibernateFactory;
+import com.redhat.rhn.domain.channel.Channel;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+import java.util.Optional;
+
+public class HubFactory extends HibernateFactory {
+
+    private static final Logger LOG = LogManager.getLogger(HubFactory.class);
+
+    /**
+     * Constructor
+     */
+    public HubFactory() {
+        // TODO document why this constructor is empty
+    }
```

Comment:
Since there is currently nothing to initialize, this constructor can be removed.